cli/lib/command/internal_command_commit.dart

```diff
@@ -79,6 +81,21 @@ class InternalCommandCommit extends InternalCommand {
         "Branch name was sanitized to '$resultingBranchName'.",
       )) return;
     }
+    if (context.isBranchCreated(resultingBranchName)) {
```

Comment:
> There is easier way to check if branch is created and pointing at current comit:
> 
> ```
> git branch --show-current
> ```
> 
> Please migrate it to use this command.

![image](https://github.com/user-attachments/assets/523af4b8-f54b-456d-9144-bfd3550439fe)
gpt saying not possible to check both condition with this one command

Reply:
This is quite a weak argument; GPT only pretends it knows something, but actually, it is tripping. You shouldn't use it unless you are an expert in the topic you are asking GPT about. Please do not do this again. 

If you ask it to provide a situation when `git branch --show-current` branch would not point at the current branch - it will fail to provide that situation.